### ISSUE TITLE
Testing code question

### ISSUE THREAD

**Issue by tiffany61706** [sunghoonim/DPSNet#11]
Dear author:
 
Thank you for providing such a great job. I was wondering that you set the maximum depth to 64 in the training code; however, in the test code, you set it to 10. 
Is there any reason for your settings?

![擷取](https://user-images.githubusercontent.com/39460180/57518479-a802ad80-734b-11e9-845d-d560c9c7f078.PNG)

**Comment by sunghoonim** [comment]
The maximum depth in the training code set to be 32 (mindepth * nlabel), which allow to utilize the whole depth range. In test code, the maxdepth is the user-defined parameter, so user can set any number that user wants to measure. There were no specific reason that I set 10. Only reason that I set as  10m is that maximum ground truth depth in SUN3D, RGBD, MVS are less than 10m. And, there is no quantitative results changes in SUN3D, RGBD, MVS (Small changes in Scenes11), whether we set the maximum as 10m or 32m.
